File: FinalCapstoneProject/Frontend/angular/src/app/model/cuisine.ts
export class Cuisine
{
    cuisineId:string;
    cuisineName:string;
    cuisineDescription:string;
    price:number;
    qty:number;
    image:any;

    constructor(cuisineId:any, cuisineName:any, cuisineDescription:any, price:any,qty:any, image:any)
    {   
        this.cuisineId = cuisineId;
        this.cuisineName = cuisineName;
        this.cuisineDescription = cuisineDescription;
        this.price = price;
        this.qty=qty;
        this.image = image;
    }
}

File: Backend/FoodieApp/restaurantService/src/main/java/com/example/restaurantService/Model/Cuisine.java
package com.example.restaurantService.Model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document
@NoArgsConstructor
@AllArgsConstructor
@Data
public class Cuisine
{
    @Id
    private String cuisineId;
    private byte[] image;
    private String cuisineName;
    private String cuisineDescription;
    private int price;
    private int qty;
}
